src/Identity/Identity.php

```diff
@@ -2,103 +2,120 @@
 
 namespace ID3Global\Identity;
 
-class Identity {
+use ID3Global\Identity\Address\AddressContainer;
+use ID3Global\Identity\Documents\DocumentContainer;
+
+class Identity
+{
     /**
      * @var string The const returned by the ID3Global API when this identity passed verification, according to the
-     * ruleset used.
+     *             ruleset used.
      */
-    const IDENTITY_BAND_PASS = 'Name, Address & DOB Match';
+    const IDENTITY_BAND_PASS = 'PASS';
```

Comment:
I'm intrigued by this change - by any chance have you discussed with GPG how the 'BandText' field is handled? For my use case, the BandText we get back is the old value (`Name, Address & DOB Match`. But their documentation alternatively states `PASS` or `Pass`.

I note there are other forks of this module that also change this to PASS, so it might be an idiosyncrasy of the specific implementation GPG has done for me - but if so, we might have to expose this as a configuration option to describe what a 'passing' BandText should be.

Nothing for you to change here either way, I'm just interested to see what you've seen yourself.

Thanks!

Reply:
By default, it is `PASS` for successful responses. But you are free to change it to any string in the ID3global's admin page.

Reply:
I do not think this should be a part of the package, since this is a user-defined property not part of the API spec. Clients of the package can use whatever they want to compare the responded band text against.

Reply:
Now I see it may be useful for clients to test their code when they use `GlobalAuthenticationGatewayFake`.

Reply:
@madmatt , I have moved the band text values to the `GlobalAuthenticationGatewayFake` class. Individual users of the package can(should) define their own constants according to the set rules and use them instead to check the response status.